.github/workflows/ci.yml

```
@@ -9,7 +9,7 @@ jobs:
 
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu
```

```suggestion
    runs-on: ubuntu-latest
```

whoops bad find/replace